src/EducationService/appsettings.Production.json

```diff
@@ -0,0 +1,10 @@
+﻿{
+  "Serilog": {
+  },
+  "ConnectionStrings": {
+    "SQLConnectionString": "Server=127.0.0.1,20340;User=sa;Password=Education_1234;Database=EducationDB;"
+  },
+  "RabbitMQ": {
+    "Host": "lt_do_rabbitmq"
+  }
+}
```

Comment:
maybe add CacheLive for Redis and MemoryCache?

Reply:
now we do not use memory cache, it will be possible to do it later